backstop_data/engine_scripts/puppet/price-set/delete-price-field-modal.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = async (engine, scenario, viewport) => {
+  const priceFieldsEmpty = await engine.$('#crm-main-content-wrapper .messages');
+
+  await require('./view-and-edit-price-fields')(engine, scenario, viewport);
+  if(priceFieldsEmpty) {
+    console.warn('No Price Field list present!')
+    console.log('Taking the "Price Fields List" Page screenshot..');
+  } else {
+    await engine.click('.crm-entity span.crm-hover-button');
+    await engine.click('a[title="Delete Price"]');
```

Comment:
Is this the only selector that can be used? `a[title="Delete Price"]` seems as bit unstable selector to me cause you can simply change text of the link and then it will stop working.

Reply:
Yes! But someone will only changes the title of the link if link functionality is changing too. a tag with title = Delete Price will only be used to delete the price.

And also in vanila civiCRM  there is no other selector available which can specifically selects this element .

Reply:
The very last argument is the only valid one, because you don't need to change the meaning in order to change the text itself. Examples:

`Delete Price` -> `Delete price` -> `DELETE PRICE` -> `Delete` -> `Remove Price` -> `Удалить` :)

Reply:
I checked CiviCRM HTML lacks many basic classes on elements. Which ends up in selecting elements using attribute selections only.